tools/lsp-wake/main.cpp

```diff
@@ -323,29 +320,42 @@ class LSPServer {
 
   MethodResult initialize(const JAST &receivedMessage) {
     MethodResult methodResult;
-    if (!isSTDLibValid) {
-      notifyAboutInvalidSTDLib(methodResult);  // set notification
+
+    std::string stdLibPath = wcl::make_canonical(find_execpath() + "/../../share/wake/lib");
+    auto initializationOptions = receivedMessage.get("params").get("initializationOptions");
+    if (initializationOptions.kind == JSON_OBJECT) {
+      auto stdLibPathEntry = initializationOptions.get("stdLibPath");
+      if (stdLibPathEntry.kind == JSON_STR) {
+        stdLibPath = stdLibPathEntry.value;
+      }
+    }
+
+    bool stdLibValid = is_readable((stdLibPath + "/core/boolean.wake").c_str());
+    if (!stdLibValid) {
+      notifyAboutInvalidSTDLib(methodResult, stdLibPath);  // set notification
       methodResult.response = JSONConverter::createInitializeResultInvalidSTDLib(receivedMessage);
-    } else {
-      methodResult.response = JSONConverter::createInitializeResultDefault(receivedMessage);
+      return methodResult;
```

Comment:
Wake doesn't currently function at all without a stdlib because we use things like `List` and `JValue` in our prims so you can do which ever is easiest to implement but I agree that we should do one or the other.

Reply:
alright perfect, I'll just not support it then. Which is what this PR does